qemu/tests/cfg/acpi_index.cfg

```diff
@@ -0,0 +1,29 @@
+- acpi_index:
+    virt_test_type = qemu
+    type = acpi_index
+    only Linux
+    no q35
+    clone_master = yes
+    master_images_clone = image1
+    remove_image_image1 = yes
+    nic_hotplug_count=1
+    pci_model = virtio-net-pci
```

Comment:
Changed it.

Reply:
Hi @PaulYuuu I realize that this parameter will be supported on `q35` in the future. It can test the `e1000e` driver. So I keep this usage to facilitate future changes.